src/reducers/articlesReducer.js

```diff
@@ -0,0 +1,238 @@
+import types from '../actions/actionTypes';
+
+const {
+  GET_ARTICLES, UPDATE_ACTIVE_PAGE, GET_CATEGORIES,
+  GET_TOP_RATED, GET_FEED, GET_FAVORITE_ARTICLES,
+  DISPLAY_NEXT_FAVORITE_ARTICLE,
+} = types;
+
+export const initialArticlesState = {
+  isLoading: false,
+  errors: {
+    statusCode: 0,
+    message: '',
+    response: {},
+  },
+  limit: 4,
+  activePage: 1,
+  articlesCount: 0,
+  articles: {},
```

Comment:
can rename this to featured articles. There is already an articles variable to avoid clashes